src/main/java/mServer/crawler/sender/br/BrWebAccessHelper.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * BrWebAccessHelper.java
+ * 
+ * Projekt    : MServer
+ * erstellt am: 07.10.2017
+ * Autor      : Sascha
+ * 
+ * (c) 2017 by Sascha Wiegandt
+ */
+package mServer.crawler.sender.br;
+
+import java.net.MalformedURLException;
+
+import org.apache.logging.log4j.Logger;
+
+import com.google.gson.JsonSyntaxException;
+import mServer.crawler.FilmeSuchen;
+import mServer.crawler.RunSender;
+import mServer.crawler.sender.MediathekReader;
+
+public class BrWebAccessHelper<V> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') V is not used in the class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2326)